code/backend/batch/utilities/tools/QuestionAnswerTool.py

```diff
@@ -20,31 +30,132 @@ def __init__(self) -> None:
         self.name = "QuestionAnswer"
         self.vector_store = AzureSearchHelper().get_vector_store()
         self.verbose = True
+        self.env_helper = EnvHelper()
+        self.config = ConfigHelper.get_active_config_or_default()
 
-    def answer_question(self, question: str, chat_history: List[dict], **kwargs: dict):
-        config = ConfigHelper.get_active_config_or_default()
+        self._user_prompt = """## Retrieved Documents
```

Comment:
I was unsure where to put these two prompts, I don't think they belong here, but I also don't think they should belong in the config as they need to be a specific format 🤷‍♂️